src/Root.tsx

```diff
@@ -1,12 +1,16 @@
+import '@expo/metro-runtime'
```

Comment:
not 100% sure if we need this tbh, it might be a dust of my hmr fixes tries, happy to remove it if we confirm we don't need it

Reply:
Removed!